api/src/main/java/org/openmrs/module/registrationcore/api/mpi/pixpdq/PdqPatientFetcher.java

```diff
@@ -58,7 +56,14 @@ public Patient fetchMpiPatient(String patientIdentifier) {
         }
     }
 
+    @Override
+    public Patient fetchMpiPatient(String patientIdentifier) {
+        return fetchMpiPatient(patientIdentifier, RegistrationCoreConstants.MPI_IDENTIFIER_TYPE_ECID_NAME);
```

Comment:
We should rather use the UUID of the identifier type, not the name. Do we have the UUID of ECID? If not, we can workaround for now by getting the ECID id type by name and then just passing the UUID down - this will result in a double fetch for now, which we can live with.